stdlib/Serialization/src/Serialization.jl

```diff
@@ -28,6 +29,8 @@ end
 
 Serializer(io::IO) = Serializer{typeof(io)}(io)
 
+const current_module = ScopedValue{Union{Nothing,Module}}(nothing)
```

Comment:
Why isn't this just a struct field of the Serializer?

Reply:
Ok, now I know why: there are other subtypes that won't have the field, so rather than need to add it to all of them I think the ScopedValue is better.